__tests__/pet.test.js

```diff
@@ -0,0 +1,281 @@
+        pet.fitness = 1;
+        pet.hunger = 3;
+        pet.checkUp();
+
+        expect(pet.checkUp()).toBe('I need a walk');
+    });
+
+    it('if hunger level is 5 or more, return "I am hungry"', () => {
+        const pet = new Pet('Fido');
+        pet.fitness = 5;
+        pet.hunger = 6;
+        pet.checkUp();
+
+        expect(pet.checkUp()).toBe('I am hungry');
+    });
+
+    it('if both of the above are true, return "I am hungry AND I need a walk"', () => {
+        const pet = new Pet('Fido');
+        pet.fitness = 1;
+        pet.hunger = 6;
+        pet.checkUp();
+
+        expect(pet.checkUp()).toBe('I am hungry AND I need a walk');
+    });
+
+    it('if both of the above are true, return "I feel great!"', () => {
+        const pet = new Pet('Fido');
+        pet.fitness = 5;
+        pet.hunger = 3;
+        pet.checkUp();
+
+        expect(pet.checkUp()).toBe('I feel great!');
+    });
+
+    it('throws an error if the pet is not alive', () => {
+        const pet = new Pet('Fido');
+        pet.age = 30;
+        expect(pet.checkUp()).toBe('Your pet is no longer alive :(');
+    });
+    
+});
+
+
+describe('isAlive', () => {
+
+    const pet = new Pet('Fido');
+
+    it('if fitness is 0 or less, return false', () => {
+        pet.fitness = -3;
+        pet.hunger = 5;
+        pet.age = 10;
+
+        expect(pet.isAlive).toBeFalsy();
+    });
+
+    it('if hunger is 10 or more, return false', () => {
+        pet.fitness = 3;
+        pet.hunger = 12;
+        pet.age = 10;
+
+        expect(pet.isAlive).toBeFalsy();
+    });
+
+    it('if age is 30 or more, return false', () => {
+        pet.fitness = 3;
+        pet.hunger = 5;
+        pet.age = 32;
+
+        expect(pet.isAlive).toBeFalsy();
+    });
+
+    it('if none of above, return true', () => {
+        pet.fitness = 3;
+        pet.hunger = 5;
+        pet.age = 10;
+
+        expect(pet.isAlive).toBeTruthy();
+    });
+})
+
+describe('adoptChild', () => {
+
+    const parent = new Pet('Yeppi');
+    const child1 = new Pet('Happy');
+
+    it('adopt a child with children property of array \
+    where the first element is the child instance you passed', () => {
+        parent.adoptChild(child1);
+
+        expect(parent.children).toEqual([ { name: 'Happy', age: 0, hunger: 0, fitness: 10, children: [] } ]);
+    });
+
+    it('adopt two more children, resulting in three children', () => {
+        const child2 = new Pet('Doory');
+        const child3 = new Pet('Nyangi');
+        parent.adoptChild(child2);
+        parent.adoptChild(child3);
+
+        expect(parent.children).toEqual(
+            [ 
+                { name: 'Happy', age: 0, hunger: 0, fitness: 10, children: [] },
+                { name: 'Doory', age: 0, hunger: 0, fitness: 10, children: [] },
+                { name: 'Nyangi', age: 0, hunger: 0, fitness: 10, children: [] } 
+            ]);
+    });
+
+    it('increase first child Happy\'s age by 2', () => {
```

Comment:
If you want to avoid having to escape characters like this, use double quotes to wrap the string:
```suggestion
    it("increase first child Happy's age by 2", () => {
```